### PR TITLE
[MAG-4186] Video Play button 

### PR DESCRIPTION
**Pull request by philjonasfiltered**
https://filtered-ewb.atlassian.net/browse/MAG-4186

### CODE REVIEW

**Comment by matthill8286** [comment]
@philjonasfiltered the changes made to social youtube will break the components using it if this is released. Can you revert that part and then you can merge your new icon.

File: dist/components/index.d.ts [hunk 1]
```diff
@@ -94,7 +95,7 @@ export { default as SocialFacebook } from './social/SocialFacebook';
 export { default as SocialInstagram } from './social/SocialInstagram';
 export { default as SocialLinkedin } from './social/SocialLinkedin';
 export { default as SocialTwitter } from './social/SocialTwitter';
-export { default as SocialYouTube } from './social/SocialYouTube';
+export { default as SocialYoutube } from './social/SocialYoutube';
```
Comment (matthill8286) [inline dist/components/index.d.ts:98]: you need to revert this as this creates a breaking change.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@excelwithbusiness/webmobile-svg-library",
-  "version": "0.0.51",
+  "version": "0.1.0",
```
Comment (matthill8286) [inline package.json:3]: have you just added this or did you run yarn bump??

**Review by matthill8286 — APPROVED** [review]
Approved and can be merged once you've addressed the changes.

File: dist/components/index.d.ts (hunk 1 above)
Comment (philjonasfiltered) [reply dist/components/index.d.ts:98]: sure thing. not sure why that happened though. Also, I ran `bump:minor` and now the version is at 0.1. is that ok?

File: package.json (hunk 2 above)
Comment (philjonasfiltered) [reply package.json:3]: I just asked you about that 😆  I did run bump yes. should I revert it and how?

**Pull request merged**